### DISCUSSION TITLE
Can Vorta backup to multiple smaller disks?

### DISCUSSION THREAD

**Discussion by rowatt** [borgbase/vorta#1322] [FAQ] [answered]
I need to backup a 16+TB disk array and would like to do that to a series of smaller (3 or 4 TB) disks which I have. Is this possible with Vorta?

**Comment by real-yfprojects** [comment, marked as answer]
No, it is not. But you can join the discussion about such a feature in borgbackup/borg#4896 at the borg repository which is the backend vorta uses.

**Reply by rowatt** [reply]
Thanks for clarifying, and for the link.

From borg discussion it doesn't look like one backup to multiple smaller disks is likely any time soon, if ever, but there was a link from that discussion to [DAR](http://dar.linux.free.fr) which is designed for backing up to multiple disks. That looks like it will do what I need for the time being, albeit without a nice GUI. Thanks.

**Reply by m3nu** [reply]
Not really what Borg is made to do. Something like LVM may be useful to combine them to one file system?
